src/components/Uploader/index.js

```diff
@@ -50,95 +43,219 @@ const MobileUploadArea = ({ onFileSelected, children, isPoA, translate }) =>
     </div>
   </div>
 
-const DesktopUploadArea = ({ translate, onFileSelected, error, uploadIcon, changeFlowTo, mobileFlow }) =>
-  <div className={ style.crossDeviceInstructionsContainer }>
-    <i className={ classNames(theme.icon, style.icon, style[uploadIcon]) } />
+const DesktopUploadArea = ({
+  translate,
+  onFileSelected,
+  error,
+  uploadIcon,
+  changeFlowTo,
+  mobileFlow,
+}) => (
+  <div className={style.crossDeviceInstructionsContainer}>
+    <i className={classNames(theme.icon, style.icon, style[uploadIcon])} />
     <div>
-      {!mobileFlow && // Hide for mobileFlow on desktop browser as `test` Node environment has restrictedXDevice set to false
+      {!mobileFlow && ( // Hide for mobileFlow on desktop browser as `test` Node environment has restrictedXDevice set to false
         <Button
           variants={['centered', 'primary']}
-          className={ style.crossDeviceButton }
+          className={style.crossDeviceButton}
           onClick={() => changeFlowTo('crossDeviceSteps')}
         >
-          { translate('capture.switch_device') }
-        </Button>}
-      <CustomFileInput className={ style.desktopUpload } onChange={ onFileSelected }>
-        {error && <UploadError { ...{ error, translate } } />}
-        <button type="button" className={ theme.link } data-onfido-qa="uploaderButtonLink">
-          { translate('capture.upload_file') }
+          {translate('capture.switch_device')}
+        </Button>
+      )}
+      <CustomFileInput
+        className={style.desktopUpload}
+        onChange={onFileSelected}
+      >
+        {error && <UploadError {...{ error, translate }} />}
+        <button
+          type="button"
+          className={theme.link}
+          data-onfido-qa="uploaderButtonLink"
+        >
+          {translate('capture.upload_file')}
         </button>
       </CustomFileInput>
     </div>
   </div>
+)
+
+const PassportMobileUploadArea = ({ nextStep, children, translate }) =>
+  <div className={classNames(style.uploadArea, style.uploadAreaMobile)}>
+    { children }
+    <div className={style.buttons}>
+      <Button
+        variants={['centered', 'primary']}
+        className={style.button}
+        onClick={nextStep}
+      >
+        {translate('capture.take_photo')}
+      </Button>
+    </div>
+  </div>
+
+const PassportDesktopUploadArea = ({
```

Comment:
`PassportDesktopUploadArea` has a lot of repeated code from `DesktopUploadArea`, we can probably use only one of these components by passing a child component to handle the button part

Reply:
Good spot. Will work on de-duping the code for these components.